crates/astria-celestia-client/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "astria-celestia-client"
+description = "an extension of eigerco's celestia client with astria specific pieces"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[dependencies]
+async-trait = { workspace = true }
+base64 = { workspace = true }
+base64-serde = "0.7.0"
+ed25519-consensus = { workspace = true }
+serde = { workspace = true, features = ["derive"] }
+serde_json.workspace = true
+sha2 = "0.10"
+tendermint = { workspace = true }
+thiserror = { workspace = true }
+
+sequencer-validation = { package = "astria-sequencer-validation", path = "../astria-sequencer-validation" }
+sequencer-types = { package = "astria-sequencer-types", path = "../astria-sequencer-types" }
+
+### Celestia specific imports
+#
+# The crates imported here are all specific to eigerco's celestia APIs
+# (including jsonrpsee).
+# As none of them are reexported it is important to keep these in sync
+# when updating.
+jsonrpsee = { version = "0.20", features = ["client-core", "macros"] }
```

Comment:
should we try to upstream to have them re-export?

Reply:
I opened this:
https://github.com/eigerco/celestia-node-rs/issues/100
Probably makes sense to just submit PRs ourselves, yeah.